polyfills/Object/values/polyfill.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+	function values(object) {
+		return Object.keys(object).map(function (key) {
+			return object[key];
+		});
+	}
+
+	Object.defineProperty(Object, 'values', {
+		configurable: true,
+		enumerate: false,
+		value: values,
```

Comment:
You could inline the function anonymously here to avoid creating the overhead of an additional symbol.